.github/workflows/dependency-scans.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Dependency Scans
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+    paths-ignore:
+      - 'docs/**'
+  schedule:
+    - cron: '35 18 * * 3'
+
+jobs:
+  bundle-audit:
+    name: Bundle audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+      - name: Install dependencies
```

Comment:
[question] do we want to add \n between these steps to make it consistent? i can also see wanting to group them together to make it clear these steps are related

Reply:
Good question - I like having all the "boilerplate action setup" steps bunched together and separated from the "actual value generated here" steps. But I'm open to other opinions

Reply:
heh yeah i guess -- whatever we choose can it just be consistent? i'm happy to keep the boilerplate steps bunched together. is it possible even to maybe pull those out and put them in an action helper or something?

Reply:
Interesting - I'm going to look into action helpers.